reference/glossary.md

```diff
@@ -25,14 +23,14 @@ enclosing [core](#core) itself as the subject.
 There are two kinds of arms: _dry_ and _wet_. Most arms are
 dry.
 
-- ##### dry:
+##### dry
```

Comment:
Don't we want to nix the h5?

Reply:
I didn't take the liberty to change the header depth in this document. Would prefer it be merged ASAP and we can fix that in a separate batch.